docs/user/tutorial/quickstart/intellij.rst

```diff
@@ -14,7 +14,7 @@ Pre-Requisites
 --------------
 This guide assumes the following:
 
-* You have the latest JDK installed (8 at the time this article was written) installed. If not the TODO Eclipse Quickstart
+* You have the latest JDK installed (8 at the time this article was written) installed. If not the `Eclipse Quickstart <./eclipse.html>`_
```

Comment:
Java 8 is no longer the latest at the time of writing

Reply:
That was mostly just a fix to the TODO; all the quickstarts should probably have updated wordings around java versions